Content.Server/Botany/Systems/PlantHolderSystem.cs

```diff
@@ -441,29 +441,24 @@ public void Update(EntityUid uid, PlantHolderComponent? component = null)
             component.Health -= 6 * healthMod;
         }
 
-        // Make sure the plant is not starving.
-        if (_random.Prob(0.35f))
+        if (component.SkipAging < 10)
         {
+
+            // Make sure the plant is not starving.
             if (component.NutritionLevel > 5)
             {
-                component.Health += healthMod;
+                component.Health += Convert.ToInt32(_random.Prob(0.35f)) * healthMod;
             }
             else
             {
                 AffectGrowth(uid, -1, component);
                 component.Health -= healthMod;
             }
 
-            if (component.DrawWarnings)
-                component.UpdateSpriteAfterUpdate = true;
```

Comment:
Why was this removed?

Reply:
It was removed because there is a duplicate within the same if block on line 469-470.

Reply:
Aha, this wasn't clear to me just looking at the diff. Looking at the change locally, this makes sense.